lab2.js

```diff
@@ -50,24 +50,49 @@ function assert(expression, failureMessage) {
  TODO: Then, use a loop to calculate how long it took the blob to finish
  with Dowington. */
 
+function Blob(name) {
+  this.name = name;
 
-var hoursSpentInDowington; // TODO: assign me the value of the
+}
+
+var blob = new Blob('blob');
+
+var peopleEaten = 0;
+var hour = 0;
+
+for (hour; peopleEaten <= 1000; hour++) {
+  peopleEaten = hour + peopleEaten;
+  if (peopleEaten >= 1000) {
+    break;
+  }
+}
+
+var hoursSpentInDowington = hour; // TODO: assign me the value of the
                            // above calculation (how long it took
                            // the blob to eat Dowington)
 
-// Now, write a method that takes a population for an arbitrary
-// town, and the starting consumption rate, and returns the number
-// of hours the blob needs to ooze its way through that town.
+// // Now, write a method that takes a population for an arbitrary
+// // town, and the starting consumption rate, and returns the number
+// // of hours the blob needs to ooze its way through that town.
 
-function hoursToOoze(population, peoplePerHour) {
-  // TODO: implement me based on the instructions above.
-  // Be sure to then assign me to the Blob's prototype.
-}
+Blob.prototype.hoursToOoze = function(population, peoplePerHour) {
+  var hour = 0;
+
+  for (hour; population > 0; hour++) {
```

Comment:
great way to frame the problem. All the info you need, and none of it that you don't.